### ISSUE TITLE
Unable to run functions due to an issue with dplyr(?)

### ISSUE THREAD

**Issue by lduverglas99** [RobinDenz1/contsurvplot#6]
Hello!

I've been running into some issues trying to use the curve_cont and plot_surv_contour functions from the package. Specifically, whenever I try to specify my dataframe it returns with,

"Error in `dplyr::all_of()`:
! Can't subset columns that don't exist.
✖ Column `MSEE` doesn't exist.
Backtrace:
  1. contsurvplot::curve_cont(...)
 25. dplyr::all_of(needed_covars)".

I am unsure of why this occurs, whenever I use data = X for my cox proportional hazard models (or any other model really) it works just fine. It doesn't make sense to me that dplyr is trying to subset columns. 

I'm hoping to apply the contour visualization of the cox proportional hazard model to some ecological data I have. My model has a number of covariates (both continuous and categorical) and I'd like to represent each covariate as a separate contour.

Thanks!

**Comment by RobinDenz1** [comment]
I am unfortunately not sure what exactly the problem is. Could you perhaps provide a small reproducible example of the error?
If you do I will happily try to help as much as I can.

And of course you can use the same Cox model to generate multiple survival contour plots. You just have to call the `plot_surv_contour` function once for each variable of interest, adjusting the `variable` argument each time.

**Comment by lduverglas99** [comment]
Hi Robin,

Thank you for the speedy response, I apologize for the delay in my own response. I figured out what the issue is, which I'm afraid has nothing to do with your package. 

Here's my model: 
        s1 <- coxph(Surv(timeto.status,status) ~ zEggheight + zClusters + zEgglay + zOrientation +herbivory+willow+pond + relevel(MSEE$treatment, ref="Control"), data=MSEE, x=TRUE)

I now understand the reason why I keep running into that error isn't because it's having an issue loading in the data like I thought. Rather, DPLYR is having trouble with how I've tried to set the reference for the treatment category. I ended up just renaming the treatments so that control ended up being the reference. It's now all working fine!

Thank you!

**Comment by lduverglas99** [comment]
Hi Robin,

Thank you for the speedy response, I apologize for the delay in my own response. I figured out what the issue is, which I'm afraid has nothing to do with your package. 

Here's my model: 
        s1 <- coxph(Surv(timeto.status,status) ~ zEggheight + zClusters + zEgglay + zOrientation +herbivory+willow+pond + relevel(MSEE$treatment, ref="Control"), data=MSEE, x=TRUE)

I now understand the reason why I keep running into that error isn't because it's having an issue loading in the data like I thought. Rather, DPLYR is having trouble with how I've tried to set the reference for the treatment category. I ended up just renaming the treatments so that control ended up being the reference. It's now all working fine!

Thank you!
